releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml

```diff
@@ -229,63 +233,81 @@ services:
     depends_on:
       - metadata
 
-  export-client:
-    image: nexus3.edgexfoundry.org:10004/docker-export-client-go:1.1.0
+  app-service-rules:
+    image: nexus3.edgexfoundry.org:10004/docker-app-service-configurable:master
     ports:
-      - "48071:48071"
-    container_name: edgex-export-client
-    hostname: edgex-export-client
+      - "48100:48100"
+    container_name: edgex-app-service-configurable-rules
+    hostname: edgex-app-service-configurable-rules
     networks:
-      - edgex-network
+      edgex-network:
+        aliases:
+          - edgex-app-service-configurable-rules
     environment:
       <<: *common-variables
-    volumes:
-      - db-data:/data/db
-      - log-data:/edgex/logs
-      - consul-config:/consul/config
-      - consul-data:/consul/data
+      edgex_service: http://edgex-app-service-configurable-rules:48100
+      edgex_profile: rules-engine
+      Service_Host: edgex-app-service-configurable-rules
+      MessageBus_SubscribeHost_Host: edgex-core-data
     depends_on:
+      - consul
+      - logging
       - data
 
-  export-distro:
-    image: nexus3.edgexfoundry.org:10004/docker-export-distro-go:1.1.0
-    ports:
-      - "48070:48070"
-      - "5566:5566"
-    container_name: edgex-export-distro
-    hostname: edgex-export-distro
-    networks:
-      - edgex-network
-    volumes:
-      - db-data:/data/db
-      - log-data:/edgex/logs
-      - consul-config:/consul/config
-      - consul-data:/consul/data
-    depends_on:
-      - export-client
-    environment:
-      <<: *common-variables
-      EXPORT_DISTRO_CLIENT_HOST: export-client
-      EXPORT_DISTRO_DATA_HOST: edgex-core-data
-      EXPORT_DISTRO_CONSUL_HOST: dgex-config-seed
-      EXPORT_DISTRO_MQTTS_CERT_FILE: none
-      EXPORT_DISTRO_MQTTS_KEY_FILE: none
-
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:1.0.0
+    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
```

Comment:
Should this image have a version like `1.1.0`?

Reply:
See previous response